stix_shifter_modules/splunk/stix_translation/json/from_stix_map.json

```diff
@@ -83,9 +83,10 @@
     "file": {
         "cim_type": "endpoint",
         "fields": {
-            "hashes.MD5": "file_hash",
-            "hashes.'SHA-1'": "file_hash",
-            "hashes.'SHA-256'": "file_hash",
+            "hashes.MD5": ["Hashes.MD5"],
+            "hashes.SHA1": ["Hashes.SHA1"],
+            "hashes.SHA256": ["Hashes.SHA256"],
```

Comment:
please correct the hash property names. file.hashes properties must be any of these `MD5, SHA-1, SHA-256, SHA-512`